docs/2_new_data_model.md

```diff
@@ -0,0 +1,104 @@
+# 2: Train with customized datasets
+
+In this note, you will know how to train and test predefined models with customized datasets. We use the Waymo dataset as an example to describe the whole process.
+
+The basic steps are as below:
+
+1. Prepare the customized dataset
+2. Prepare a config
+3. Train, test, inference models on the customized dataset.
+
+## Prepare the customized dataset
+
+There are three ways to support a new dataset in MMDetection3D:
+
+1. reorganize the dataset into existed format.
+2. reorganize the dataset into a middle format.
+3. implement a new dataset.
+
+Usually we recommend to use the first two methods which are usually easier than the third.
+
+In this note, we give an example for converting the data into KITTI format.
+
+**Note**: We take Waymo as the example here considering its format is totally different from other existed formats. For other datasets using similar methods to organize data, like Lyft compared to nuScenes, it would be easier to directly implement the new dataset inherited from an existed one.
```

Comment:
Do you mean `it would be easier to directly implement the new dataset than inherited from an existed one.`?

Reply:
It does seem confusing. I mean `it would be easier to directly implement the new data converter instead of converting it to another format.`